### PR TITLE
Add user role check for all commands except "хочу в отпуск"

### LINKED ISSUE #1: Limit the use of the commands (except of 'хочу в отпуск') to admin purposes only

### PR DESCRIPTION
**Pull request by BehindLoader**
close #1

### CODE REVIEW

**Comment by eugulixes** [comment]
Let's limit the use of the `отменить заявку` command to admin purposes only as well.

File: src/viva.js [hunk 1]
```diff
@@ -8,9 +8,9 @@
 //
 // Commands:
 //   hubot хочу в отпуск - initiates a new leave request
-//   hubot одобрить заявку @username - approves the leave request for the specifies user
-//   hubot отклонить заявку @username - rejects the leave request for the specifies user
-//   hubot отменить заявку @username - cancels the leave request for the specifies user
+//   hubot одобрить заявку @username - approves the leave request for the specifies user (privileged: admins only)
+//   hubot отклонить заявку @username - rejects the leave request for the specifies user (privileged: admins only)
+//   hubot отменить заявку @username - cancels the leave request for the specifies user (privileged: admins only)
 //
```
Comment (polina-popova) [inline src/viva.js]: It should be "specified".

File: README.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ A Hubot script which helps users to create leave requests.
 ## Prerequisites
 
 The bot must be in the channel specified via the `LEAVE_COORDINATION_CHANNEL` environment variable.
+The bot has to have `view-full-other-user-info` permission.
```
Comment (eugulixes) [inline README.md]: Use a list here, i.e. `*`.

File: src/viva.js (hunk 1 above)
Comment (eugulixes) [reply src/viva.js]: And the word `approved` was dropped for some reason.

**Pull request merged**